sdk/src/main/java/com/mapbox/maps/StyleObserver.kt

```diff
@@ -92,6 +101,11 @@ internal class StyleObserver(
         loadStyleTransitionOptions = null
       }
     }
+    if (loadStyleEarlyStyleCallback) {
+      if (eventData.type == StyleDataType.STYLE) {
+        invokeStyleListeners()
+      }
+    }
```

Comment:
```suggestion
    if (loadStyleEarlyStyleCallback && eventData.type == StyleDataType.STYLE) {
        invokeStyleListeners()
      }
    }
```

Reply:
the `loadStyleEarlyStyleCallback` is removed so this change is not valid any more.